vital/types/local_geo_cs.cxx

```diff
@@ -59,8 +65,16 @@ write_local_geo_cs_to_file(local_geo_cs const& lgcs,
   {
     ofs << std::setprecision(12) << lon_lat_alt[1]
                                  << " " << lon_lat_alt[0]
-                                 << " " << lon_lat_alt[2];
+                                 << " " << lon_lat_alt[2]
+                                 << std::endl;
+    return ofs.good();
+  }
+  else
+  {
+    throw vital::file_write_exception(file_path,
+                                      "Unable to open file for writing.");
   }
+  return false;
```

Comment:
You'll never actually get here because you'll throw the exception first.  I'm not sure that this function should both throw an exception or return false on different types of errors.  I suggestion you just log an error message and return false.